doc/papers/joss/paper.md

```diff
@@ -7,6 +7,9 @@ tags:
   - spectral elements
   - matrix-free
 authors:
+  - name: Ahmad Abdelfattah
```

Comment:
Hi @abdelfattah83 thanks for adding yourself and welcome! Yes, you assumed right. Our first idea was to include all authors from the user-manual (who, in turn, are all authors who satisfy both points in our [CONTRIBUTING](https://github.com/CEED/libCEED/blob/master/CONTRIBUTING.md) file.), in alphabetical order by last name. However, the CU Boulder's libCEED team (cc: @LeilaGhaffari, @jeremylt ) really thinks that @jedbrown should be the first author for this first libCEED's whitepaper. So we can keep the chosen convention, but with the exception that Jed is the first author, if everybody is fine with that.

Reply:
With Jed being the architect for the library and the long term maintainer, it makes sense to me for him to be listed as first author on the first libCEED paper.

Reply:
Yes, I agree. It seems fair to me.

Reply:
I think that's a good plan.